### PR TITLE
Adding zbar-py

### PR DESCRIPTION
**Pull request by Ryan-Dmello**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin, please ping team

File: recipes/zbar-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True     # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (nehaljwani) [inline recipes/zbar-py/meta.yaml:20]: You shouldn't need 'cxx' here. The package only has a C extension, no C++ extension

File: recipes/zbar-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+    - libiconv
```
Comment (nehaljwani) [inline recipes/zbar-py/meta.yaml:27]: Why is libiconv here?

File: recipes/zbar-py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+test:
+  imports:
+    - zbar
+
+about:
+  home: "https://github.com/zplab/zbar-py"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "zbar package"
```
Comment (nehaljwani) [inline recipes/zbar-py/meta.yaml:38]: Could you please update the summary to be a bit more descriptive?

Comment (Ryan-Dmello) [reply recipes/zbar-py/meta.yaml]: Updated as per suggestion

File: recipes/zbar-py/meta.yaml (hunk 1 above)
Comment (Ryan-Dmello) [reply recipes/zbar-py/meta.yaml]: Updated as per suggestion

File: recipes/zbar-py/meta.yaml (hunk 2 above)
Comment (Ryan-Dmello) [reply recipes/zbar-py/meta.yaml]: Updated as pr suggestion

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin, please ping team

**Pull request merged**